sass/_large.scss

```diff
@@ -0,0 +1,15 @@
+@media (min-width: 1200px) {
+    nav.nav-mobile {
+        display: none;
+    }
+    button.burger-btn {
+        display: none;
+    }
+}
+
+@media (max-width: 1200px) {
+    nav.nav {
+        display: none;
+    }
+    
```

Comment:
I'm not sure what to think about you way of adding empty lines in different situations, but in this one I'm pretty sure it's redundant.